framework-cloud/framework-openshift-operator/src/main/java/org/kie/cloud/openshift/operator/scenario/builder/ClusteredWorkbenchRuntimeSmartRouterTwoKieServersTwoDatabasesScenarioBuilderImpl.java

```diff
@@ -144,17 +139,6 @@ public ClusteredWorkbenchRuntimeSmartRouterTwoKieServersTwoDatabasesScenarioBuil
         return this;
     }
 
-    @Override
-    public ClusteredWorkbenchRuntimeSmartRouterTwoKieServersTwoDatabasesScenarioBuilder withBusinessCentralMavenUser(String user, String password) {
-        kieApp.getSpec().getCommonConfig().setMavenPassword(password);
-        kieApp.getSpec().getObjects().getConsole().addEnv(new Env(ImageEnvVariables.KIE_MAVEN_USER, user));
-
-        for (Server server : kieApp.getSpec().getObjects().getServers()) {
-            server.addEnv(new Env(propertyNames.workbenchMavenUserName(), user));
-        }
```

Comment:
I'm not sure if this method is used in some scenario or not. But there was somewhere set maven user for this scenario, we should keep configuration of this user for the Kie Server, instead of mavenUser there can be put appUser.
Ignore this comment if this code is not used.

Reply:
Yes, this method was not used anywhere.